docs/conf.py

```diff
@@ -143,6 +142,15 @@
     'display_version': True,
     'prev_next_buttons_location': 'bottom',
     'style_external_links': True,
+    'top_menu_links': [
+        'qiskit_core_getting_started',
+        'qiskit_core_tutorials',
+        'qiskit_partners',
+        'qiskit_applications',
+        'qiskit_experiments',
+        'resources',
+        'qiskit_organization_github'
+    ]
```

Comment:
As the `conf.py` file is related with the docs project and not with the theme, it's a way I found to separate the content from the structure to be able to reuse the layout along on the projects instead of creating a new one for each repo. This can be a config setting that says to the theme the links it should so. It tries to reuse as much as code as we can having in mind that we need to work on the documentation reorganization. If we unify and reuse the layouts, we just have one place to change things (contrary that what we have right now that if we want to change something, we have to do it on every repository). Is it more clear now? If not, let me know and I try again! :)

Reply:
Yeah, that's kind of what I meant - the allowed links are still defined within the theme, so there's less separation here than I'd have expected.  For example, I couldn't inject a new link (e.g.) `'qiskit_metal'` into the theme from this configuration; I can only select those that are already defined.  With it being an option here, I'd have expected the setting to be a way to define new links, not just select those that exist.

If it's intended to be like that, it's ok - the alternative might involve separating out _a lot_ of data that the various components all want to reuse.

Reply:
It's intended to be like that, I have explained more [here](https://github.com/Qiskit/qiskit_sphinx_theme/pull/43#discussion_r793509792). I didn't add all links yet because I was thinking on adapt one repo at a time, starting with the [qiskit metapackage ](https://github.com/Qiskit/qiskit), so I just added the links needed on top menu on this repo.

Reply:
Could you explain me more about the alternative you are thinking?

Reply:
Jake I explained the reasoning a bit more in your other thread before I saw this one 😅 let me know if you need more clarification